### DISCUSSION TITLE
Why the process of "Performing download step (git clone) for 'hypre'" failed?

### DISCUSSION THREAD

**Discussion by RGQTJU** [amanzi/amanzi#749] [Q&A]
Hi, everyone! I'm new to ats. And I built this program follow exactly the same way provided in the installation guide. However, this process fails at the stage of "Performing download step (git clone) for 'hypre'". I tried to build hypre by myself and set the installation directory to "amanzi_tpls-install-master-Release", but the installation process stopped at the same stage. What's the problem?

**Comment by lipnikov** [comment]
Errors and other issues are logged in build/tpls/hypre/hypre-timestamps/hypre-download-*.log
Review these files first, since installation problems could be also related to system configuration.

**Comment by lipnikov** [comment]
RESOLVED
